cookbook/mirabella-genio-bulb.rst

```diff
@@ -0,0 +1,175 @@
+Mirabella Genio Bulbs
+=====================
+
+The Mirabella Genio is a Tuya based smart bulb sold by Kmart in Australia. 
+
+.. figure:: images/mirabella-genio-b22-rgbw.jpg
+    :align: center
+    :width: 50.0%
+
+Originally intended to be used with their companion app once flashed using `tuya-convert <https://github.com/ct-Open-Source/tuya-convert>`__ ESPHome generated firmware can be uploaded allowing you to control the bulbs via Home Assistant.
+
+Thanks to the `existing work <https://github.com/arendst/Sonoff-tasmota/wiki/Mirabella-Genio-Bulb>`__ by `@so3n <https://github.com/arendst/Sonoff-tasmota/wiki/Mirabella-Genio-Bulb>`__ which these adaptions created by `@cryptelli <https://community.home-assistant.io/u/cryptelli>`__ and `@bircoe <https://community.home-assistant.io/u/bircoe>`__ are based on.
+
+Dimmable Warm/Cool White Bulbs
+------------------------------
+
+.. code-block:: yaml
+
+    esphome:
+      name: mirabella_genio_cw_1
+      platform: ESP8266
+      board: esp01_1m
+
+      # Ensure the light turns on by default if the physical switch is actuated.
+      # https://esphome.io/components/esphome#esphome-on-boot
+      on_boot:
+        priority: 100 # Highest priority, ensures light turns on without delay.
+        then:
+          - light.turn_on: light
+
+    wifi:
+      ssid: 'WIFI'
+      password: 'WIFIPASS'
+
+    logger:
+
+    web_server:
+      port: 80
+
+    api:
+
+    ota:
+
+    light:
+      - platform: monochromatic
+        name: "Mirabella Genio Smart Bulb"
+        id: light
+        output: output_component1
+
+    output:
+      - platform: esp8266_pwm
+        id: output_component1
+        pin: GPIO13
+
+Tunable Warm/Cool White Bulbs
```

Comment:
Let's not use `tunable` here, the word could mean anything here. Maybe something more like cold white + warm white bulbs